trace/opencensus/trace/propagation/binary_format.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import binascii
+import collections
+import logging
+import struct
+
+from opencensus.trace.propagation import _helpers
+from opencensus.trace.span_context import SpanContext
+
+_ENABLED_BITMASK = 1
+
+# Used for decoding hex bytes to hex string.
+UTF8 = 'utf-8'
+
+VERSION_ID = 0
+TRACE_ID_FIELD_ID = 0
+SPAN_ID_FIELD_ID = 1
+TRACE_OPTION_FIELD_ID = 2
+
+# Sizes are number of bytes.
+ID_SIZE = 1
```

Comment:
Can you rename this to `VERSION_ID_SIZE`?

Reply:
This is actually the size for the version_id and the field_ids.